src/toggle-switch/toggle-switch.scss

```diff
@@ -111,11 +117,18 @@
   &:checked {
     background-color: var(--iui-color-background-primary);
     border-color: var(--iui-color-background-primary);
+    @media (forced-colors: active) {
+      background-color: ActiveText;
+      border-color: ActiveText;
```

Comment:
I think you are misusing this color as it's meant for active link text (foreground). It's too heavy for use in the background. I would just keep the default background, and use `CanvasText` for the handle.
![image](https://user-images.githubusercontent.com/9084735/163016554-1a0db394-3639-4d46-b623-8fc5b2215658.png)

Reply:
I'm seeing some UIs do that. The one downside to that is not knowing which one is checked / unchecked.
![Screen Shot 2022-04-12 at 3 22 48 PM](https://user-images.githubusercontent.com/849817/163038883-ea201bdc-6eca-4b44-961b-d96171ef6b92.png)

Microsoft appears to be using `background-color: highlight` on [this page](https://myaccount.microsoft.com/settingsandprivacy/privacy):
![Screen Shot 2022-04-12 at 3 27 08 PM](https://user-images.githubusercontent.com/849817/163039093-30467fb5-8941-4668-84a6-13d92fc5dc7c.png)
![Screen Shot 2022-04-12 at 3 27 13 PM](https://user-images.githubusercontent.com/849817/163039095-fcc76e3d-dd16-434f-b060-118a65ab0e64.png)

Reply:
Just confirmed that toggle styling is identical within Windows 10 High Contrast settings menu as well.